nl_NL/LC_MESSAGES/discordbot.po

```diff
@@ -10,27 +10,27 @@ msgstr ""
 
 # Username was given a Nitro Booster Ticket...
 msgid "BOOST_TICKET_SUCCESS"
-msgstr "%s was given a Nitro Booster Ticket, redeemable for a month of Sudomemo Plus and a special trophy. Hooray!"
+msgstr "%s Kreeg een nitro boost ticket voor een maand van sudomemo! Hoera!"
 
 # Account linking
 msgid "DISCORD_LINKING_GET_STARTED"
-msgstr "Please visit %s to get started!"
+msgstr "Bekijk %s om te beginnen!"
 
 msgid "DISCORD_LINKING_REQUIRED"
-msgstr "You'll need to link your Sudomemo account to do that."
+msgstr "Je moet je sudomemo account koppelen om dit te doen!"
 
 msgid "DISCORD_LINKING_SUCCESS"
-msgstr "Your account was successfully linked!"
+msgstr "Je account is succsessvol gekoppeld!"
 
 msgid "DISCORD_LINKING_UNLINKED"
-msgstr "The link between your Discord and Sudomemo account has been removed."
+msgstr "De koppeling van je account en discord is verwijderd."
```

Comment:
This doesn't appear to mention that the link to the Sudomemo account is what's being removed, just an account, also the first letter of "Discord" should be capitalized.